packages/qwik-city/src/buildtime/runtime-generation/generate-service-worker.ts

```diff
@@ -38,6 +38,9 @@ export function prependManifestToServiceWorker(
     return null;
   }
 
+  // TODO: add dynamic imports that don't import routes
+  // (anything that doesn't contain _hw export)
+
```

Comment:
@shairez do you want to keep that `TODO` here or should we open a new issue for it? Considering caching the dynamic imports is per se an additional improvement to the (current) prefetching.

Reply:
oh well I guess it will stay here for now :sweat_smile:

Reply:
yeah 😅

I'll remove this comment in a different PR